python/examples/basic/Getting_Started.ipynb

```diff
@@ -438,9 +449,17 @@
     "    - [Inspecting Profiles](https://whylogs.readthedocs.io/en/stable/examples/basic/Inspecting_Profiles.html) - A deeper dive on the metrics generated by whylogs\n",
     "    - [Schema Configuration for Tracking Metrics](https://whylogs.readthedocs.io/en/stable/examples/basic/Schema_Configuration.html) - Configure tracking metrics according to data type or column features\n",
     "    - [Data Constraints](https://whylogs.readthedocs.io/en/stable/examples/basic/Metric_Constraints.html) - Set constraints to your data to ensure its quality\n",
+    "    - [Merging Profiles](https://whylogs.readthedocs.io/en/stable/examples/basic/Merging_Profiles.html) - Merge your profiles logged across different computing instances, time periods or data segments\n",
     "- Integrations\n",
+    "    - [WhyLabs](https://whylogs.readthedocs.io/en/stable/examples/integrations/writers/Writing_to_WhyLabs) - Monitor your profiles continuously with the WhyLabs Observability Platform\n",
     "    - [Pyspark](https://whylogs.readthedocs.io/en/stable/examples/integrations/Pyspark_Profiling.html) - Use whylogs with pyspark\n",
-    "    - [Writing Profiles](https://whylogs.readthedocs.io/en/stable/examples/integrations/Writing_Profiles.html) - See different ways and locations to output your profiles"
+    "    - [Writing Profiles](https://whylogs.readthedocs.io/en/stable/examples/integrations/Writing_Profiles.html) - See different ways and locations to output your profiles\n",
+    "    - [Flask](https://whylogs.readthedocs.io/en/stable/examples/integrations/flask_streaming/flask_with_whylogs.html) - See how you can create a Flask app with whylogs and WhyLabs integration\n",
+    "    - [Feature Stores](https://whylogs.readthedocs.io/en/stable/examples/integrations/Feature_Stores_and_whylogs.html) - Learn how to log features from your Feature Store with feast and whylogs\n",
+    "    - [BigQuery](https://whylogs.readthedocs.io/en/stable/examples/integrations/BigQuery_Example.html) - Profile data queried from a Google BigQuery table\n",
+    "    - [MLflow](https://whylogs.readthedocs.io/en/stable/examples/integrations/Mlflow_Logging.html) - Log your whylogs profiles to an MLflow environment\n",
+    "\n",
+    "Or go to the [examples page](https://whylogs.readthedocs.io/en/stable/examples.html) for the complete list of examples!"
```

Comment:
Why are we leading users away from our github repo when they are exploring the example notebooks? The html versions here are less useful than the ipynb files  since you can't run them.

Also some of these links are broken: https://whylogs.readthedocs.io/en/stable/examples/integrations/Writing_Profiles.html

Reply:
If users are seeing this on the documentation, then we'd be leading them away from the docs page. Maybe that's ok?

Another issue is that notebooks are not well rendered on github (a lot of plots and graphs goes missing when viewing them on github), which doesn't happen on readthedocs. Referencing directly to the repo would also point to the "latest" examples, instead of the stable ones, but maybe it's not an issue as well.

About the fact that you can't run them, I think the colab button kind of takes care of it? you can run it from docs page or gh repo on collab.

Reply:
You are right, the visuals are better in the html version, and the colab button works there, good points.

Maybe it is fine. I find switching between the two locations a bit confusing. I would prefer as a user that the example notebooks in github would link to other example notebooks in github (if logically equivalent content is in more than one location). If that isn't easy to do then maybe this is an acceptable trade off to link into the read the docs here.

But we at least need to fix the links so they don't give us 400 status responses.

Reply:
Fixed the links!

Just pasting from slack for reference:

I actually agree with you on it being confusing switching between places. I don't have an easy way to do that for the moment, so I'll keep the PR as is, and try to think of a solution to change that later!